docs/user/tutorial-asgi.rst

```diff
@@ -967,6 +967,49 @@ adding ``--cov-fail-under=100`` (or any other percent threshold) to our
     tests in multiple environments would most probably involve running
     ``coverage`` directly, and combining results.
 
+Debugging ASGI Applications
+---------------------------
+(This section also applies to WSGI applications)
+
+While developing and testing ASGI applications, understanding how to configure and utilize logging can be helpful, especially when you encounter unexpected issues or behaviors.
+
+By default, Falcon does not set up logging for you, but Python's built-in `logging` module provides a flexible framework for emitting and capturing log messages. Here's how you can set up basic logging in your ASGI Falcon application:
+
+.. code:: python
+
+    import logging
+
+    logging.basicConfig(
+        format='%(asctime)s [%(levelname)s] %(message)s',
+        level=logging.INFO
+    )
+
+    logger = logging.getLogger(__name__)
+
+You can now use `logger` to log messages within your application. For example:
+
+.. code:: python
+
+    logger.info('Info message')
+    logger.warning('Warning message')
+    logger.error('Error message')
+
+It's especially useful to log exceptions and error messages that can help diagnose issues:
```

Comment:
This is a good intro on logging, but IMHO the reader of this second tutorial is already expected to be an experienced Python developer, so maybe we could shorten this generic section a bit, and tell more about Falcon's default `Exception` handler?

In addition to rendering an HTTP 500 response, it does log a message to the `falcon` logger. Maybe we could even show how this looks like?

Reply:
Yeah totally makes sense. Is something like this what you have in mind?

```
import falcon
import logging

logging.basicConfig(level=logging.INFO)

class ErrorResource:
    def on_get(self, req, resp):
        raise Exception("Something went wrong!")

app = falcon.App()
app.add_route('/error', ErrorResource())
```